lib/sasl/sasl_anonymous.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+var Builder = require('buffer-builder'),
+    Promise = require('bluebird');
+
+function SaslAnonymous () {}
+
+SaslAnonymous.prototype.getInitFrameContent = function() {
+  return new Promise(function(resolve) {
```

Comment:
nit: in my experience it's cleaner doing `Promise.resolve(value)` rather than newing a Promise if you don't need to wrap any asynchronous operations

Reply:
Fair enough. i'm not very familiar with Promises (we don't use them) so i'm glad to get this type of feedback. thanks!